lib/IAMClient.js

```diff
@@ -349,6 +349,36 @@ class IAMClient {
     }
 
     /**
+     * A getter for account canonical IDs given a list of account IDs
+     * @param{String[]} accountIds - list of account IDs
+     * @param {Object} options - additional arguments
+     * @param {string} options.reqUid - the request UID
+     * @param {werelogs.Logger} [options.logger] - Logger instance
+     * @param{Function} callback - the callback handling the array of objects
+     * and the error, if there is one
+     * @returns{undefined}
+     */
+    getCanonicalIdsByAccountIds(accountIds, options, callback) {
+        assert(Array.isArray(accountIds), 'accountIds are required');
```

Comment:
should you also assert all the accountIds are strings? Idk where they're coming from, so I'm not sure if it's an issue